data/mods/Magiclysm/items/alchemy_items.json

```diff
@@ -31,5 +31,74 @@
     "to_hit": -2,
     "qualities": [ [ "BUTCHER", -50 ] ],
     "//": "Potion recipes that require a full owlbear egg could also instead require larger amounts of stone shell. Since 1 egg gives 3 shells, you could perhaps have 1 egg be equivalent to 5 or more shells when making potions."
+  },
+  {
+    "id": "glow_dust",
+    "type": "GENERIC",
+    "symbol": "*",
+    "color": "yellow",
+    "name": "glow dust",
+    "description": "The powdered remains of a will-o-wisps's phsyical form.  It seems to still possess an otherworldly glow.",
+    "material": "powder",
+    "weight": 5,
+    "volume": 1
+  },
+  {
+    "id": "glow_light",
+    "type": "GENERIC",
+    "category": "tools",
+    "name": "magical reading light",
+    "description": "Powered by the magic of glow powder and lesser mana potions, this extremely expensive little light will provide just enough light to read by for at least a decade.  Use it to close the cover and hide the light.",
+    "weight": 270,
+    "volume": 1,
+    "price": 125000,
+    "material": "glass",
+    "symbol": ",",
+    "color": "light_blue",
+    "use_action": {
+      "target": "glow_light_off",
+      "msg": "You close the nightlight's cover.",
+      "menu_text": "Close cover",
+      "type": "transform"
+    },
+    "flags": [ "LIGHT_5", "ALLOWS_REMOTE_USE" ]
+  },
+  {
+    "id": "glow_light_off",
+    "type": "GENERIC",
+    "category": "tools",
+    "name": "magical reading light (covered)",
+    "name_plural": "magical reading lights (covered)",
+    "description": "Powered by the magic of glow powder and lesser mana potions, this extremely expensive little light will provide just enough light to read by for at least a decade.  The cover is closed.  Use it to open the cover and show the light.",
+    "weight": 270,
+    "volume": 1,
+    "price": 125000,
+    "material": "glass",
+    "symbol": ",",
+    "color": "light_blue",
+    "use_action": { "target": "glow_light", "msg": "You open the nightlight's cover.", "menu_text": "Open cover", "type": "transform" },
+    "flags": [ "ALLOWS_REMOTE_USE" ]
+  },
+  {
+    "id": "bulette_plate",
+    "type": "GENERIC",
+    "symbol": ")",
+    "color": "light_gray",
+    "name": "bulette plate",
+    "description": "The great plates from behind a bulette's head have always been prized for use in shield and armor making.",
+    "material": "leather",
```

Comment:
hmm, possibly you want to use some other material other than leather? you can add a new material for this "magical beast plating" or maybe specifically for each creature; use your judgement here

Reply:
Yeah I was having a real tough time picking a material. It is effectively a type of leather which is why I ended up there but it does seem a bit off. A "magical plating" or something, sort of the animal equivalent of superalloy, might be pretty useful for future applications too come to think of it.